src/xrCDB/Frustum.cpp

```diff
@@ -106,40 +106,40 @@ bool CFrustum::testSphere_dirty(const Fvector& c, float r) const
     case 12:
         if (planes[11].classify(c) > r)
             return FALSE;
-    case 11:
+    [[fallthrough]]; case 11:
```

Comment:
1. Code style
2. It's ok to not insert an empty line here
```suggestion
        [[fallthrough]];
    case 11:
```